src/include/usr/hwpf/hwp/mvpd_accessors/getMBvpdAttr.H

```diff
@@ -227,6 +227,13 @@ namespace getAttrData
         uint8_t              iv_systemType;
         uint8_t              iv_systemType_ext;
         uint8_t              iv_dataVersion;
+    public:
```

Comment:
Surprised these changes were required, happen to remember what failed before you changed them?